styles/styles.css

```diff
@@ -321,11 +453,88 @@ body {
   background: #dfe1e6;
 }
 
+/* Desktop - skills */
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] Same as here

Reply:
Hi @joskalenda, I have moved the media queries to a single block.